smoke/helpers.go

```diff
@@ -9,13 +10,34 @@ import (
 	. "github.com/onsi/gomega/gexec"
 )
 
+var DEFAULT_TIMEOUT = 30 * time.Second
+
+func SetBackend(appName string) {
+	config := GetConfig()
+	if config.Backend == "diego" {
+		EnableDiego(appName)
+	} else if config.Backend == "dea" {
+		DisableDiego(appName)
+	}
+}
+
 func EnableDiego(appName string) {
```

Comment:
The Windows test case requires diego. That was the reason to put EnableDiego there. Otherwise we would have dependend config settings (if you enabled windows testing, you have to enable diego too).